src/gateway/understanding-kong/how-to/aws-secrets-manager.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Securing Kong Gateway database credentials with AWS Secrets Manager"
+description: "How-to keep your {{site.base_gateway}} database credentials secure using AWS SecretsManager and Kong vault integrations."
+---
+
+Application secrets include sensitive data like passwords, keys, certifications, tokens, and other items
+which must be secured. [{{site.base_gateway}}](/gateway/{{page.kong_version}}/) supports
+[Secrets Management](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/) 
+that allows you to store secrets in various secure backend systems helping you protect them from accidental
+exposure.
+
+Traditionally, {{site.base_gateway}} is configured with static credentials for connecting 
+to its external database. This guide will show you how to configure {{site.base_gateway}} to use 
+[AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access.html) to 
+read database credentials securely instead the conventional file or environment variable based solutions.
+
+For this guide you are going to run the PostgreSQL and {{site.base_gateway}} locally on Docker. 
+You will create a secret in the AWS Secrets Manager and deploy {{site.base_gateway}} using a vault reference
+to read the value securely.
+
+### Prerequisites
+
+* It is assumed you have access to an [AWS account](https://aws.amazon.com/) and have the 
+  [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) 
+  installed and configured.
+  
+  Your AWS account or assumed role must have proper IAM permissions to allow 
+  access to the AWS Secrets Manager service. Permission policy examples can be found in the 
+  [AWS Secrets Manager documentation](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access_examples.html).
+  In order to create a new secret with the AWS CLI, you will need the following permissions:
+  * `secretsmanager:CreateSecret`
+  * `secretsmanager:PutSecretValue`
+
+  Additionally, {{site.base_gateway}} will connect to Secrets Manager and retrieve the secret value, 
+  this requires the `secretsmanager:GetSecretValue` permission.
+
+  You must be able to configure the gateway environment with the well known 
+  [AWS CLI environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
+  as this is the method {{site.base_gateway}} uses to connect to the Secrets Manager service.
```

Comment:
```suggestion
* An [AWS account](https://aws.amazon.com/). Your account must have the proper IAM permissions to allow access to the AWS Secrets Manager service. Permission policy examples can be found in the [AWS Secrets Manager documentation](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access_examples.html). Additionally, you must have the following permissions:
  * `secretsmanager:CreateSecret`
  * `secretsmanager:PutSecretValue`
  * `secretsmanager:GetSecretValue`
* The [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed and configured. You must be able to configure the gateway environment with [AWS CLI environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) because this is the method that {{site.base_gateway}} uses to connect to the Secrets Manager service.
```